govcd/org_test.go

```diff
@@ -882,3 +882,23 @@ func (vcd *TestVCD) Test_OrgGetVdc(check *C) {
 	}
 	vcd.testFinderGetGenericEntity(def, check)
 }
+
+// Tests VDC retrieval by name, by ID, and by a combination of name and ID
+func (vcd *TestVCD) Test_GetTaskList(check *C) {
+
+	if vcd.config.VCD.Org == "" {
+		check.Skip("Test_GetTaskList: Org name not given.")
+		return
+	}
+	// we need to have Tasks
+	if vcd.skipVappTests {
+		check.Skip("Skipping test because vApp wasn't properly created")
+	}
+	org, err := vcd.client.GetOrgByName(vcd.config.VCD.Org)
+	check.Assert(err, IsNil)
+	check.Assert(org, NotNil)
+
+	taskList, err := org.GetTaskList()
+	check.Assert(err, IsNil)
+	check.Assert(len(taskList.Task), Not(Equals), 0)
```

Comment:
Please add a test to check that the first task is properly formed, i.e. that it is not null and it has at least some of the expected fields, such as ID, type, owner, status, progress

Reply:
added